### PR TITLE
Added th entries up until 2024-06-21

### PR DESCRIPTION
**Pull request by h3ssto**
Added new references and theses.

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -184,6 +184,38 @@ @String{Bachelor
...
+	note = {To appear}
+}
+
+@inproceedings{HSST:SPLC24,
+	title = {{Towards Deterministic Compilation of Binary Decision Diagrams From Feature Models}},
+	author = {Heß, Tobias and Semmler, Sean and Sundermann, Chico and Thüm, Thomas},
+	booktitle = SPLC,
+	publisher = ACM,
+	address = NY,
+	note = {{To appear.}},
```
Comment (pmbittner) [inline literature.bib:205]: Double brackets and `.` intended?

File: literature.bib [hunk 2]
```diff
@@ -1906,6 +1938,20 @@ @inproceedings{WKR:SPLC22
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2021 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{FLP:VaMoS21,
```
Comment (pmbittner) [inline literature.bib:1941]: `FLP+:VaMoS21`

**Review by pmbittner — CHANGES_REQUESTED** [review]

File: literature.bib (hunk 2 above)
Comment (h3ssto) [reply literature.bib]: Fixed.

File: literature.bib (hunk 1 above)
Comment (h3ssto) [reply literature.bib]: Fixed.

File: literature.bib [hunk 3]
```diff
@@ -184,6 +184,38 @@ @String{Bachelor
...
+	note = {To appear},
+	month = SEP,
+	year = 2024
+}
+
+@inproceedings{HOB+:MODEVAR24a,
+	author = {Heß, Tobias and Ostheimer, Lukas and Betz, Tobias and Karrer, Simon and Schmidt, Tim Jannik and Coquet, Pierre and Semmler, Sean and Thüm, Thomas},
+	title = {{variability.dev: Towards an Online Toolbox for Feature Modeling}},
+	booktitle = MODEVAR,
+	note = {{To appear.}},
```
Comment (pmbittner) [inline literature.bib:214]: Sorry, there is also double brackets and . here.

**Review by pmbittner — APPROVED** [review]

File: literature.bib (hunk 3 above)
Comment (h3ssto) [reply literature.bib]: Fixed too.

**Pull request merged**